.changeset/wet-readers-join.md

```diff
@@ -0,0 +1,30 @@
+---
+'astro': minor
+---
+
+Shipped a new SSR mode, called `split`. 
+When enabled, Astro will emit a file for each page, which will render one single page.
+
+These files will be emitted inside `dist/pages`, and they will look like this:
```

Comment:
```suggestion
These files will be emitted inside `dist/pages`, mirroring your page files in `src/pages/`, for example:
```
This is the wording we went with in docs, and is more descriptive than "they will look like this" so I prefer something like this!

Reply:
Mirroring is not technically correct because the name of the files will look different. The only thing that technically we mirror is the **directory structure**. The name of the file will be different, e.g.: `blog.astro` -> `entry.blog.astro.mjs`

Reply:
OK, what about something like "mirroring the directory structure of your page files in `src/pages/`"... ?